interfaces/mount/backend.go

```diff
@@ -92,13 +93,14 @@ func (b *Backend) Remove(snapName string) error {
 // deriveContent computes .fstab tables based on requests made to the specification.
 func deriveContent(spec *Specification, snapInfo *snap.Info) map[string]*osutil.FileState {
 	// No entries? Nothing to do!
-	if len(spec.mountEntries) == 0 {
+	entries := spec.MountEntries()
+	if len(entries) == 0 {
```

Comment:
What happens if we had entries before and we don't have them now, is that a possible scenario? In such case shouldn't we let it continue and create FileState and synchronize content?

Reply:
It will happen automatically. This is how the ensure dir state logic works, if you want to have a file, describe it, if you don't describe it it will be removed.

Reply:
Isn't `len(entires) ==0 .. ` preventing this? I don't see DirState being used here?

Reply:
It's fine, the return value is used by `Setup()` if we return nil then it's just an empty map of files we want to have there.